yoh-notes

```diff
@@ -17,4 +17,60 @@ preparing an image-- running out of space
 
 install reproman on discovery
 
+2023-10-05 -- some notes. more highlighted in local pdf, heh
 
+- hard to read since often dwells on something only loosely related to the point to deliver or just sounding "sophisticated"
+
+- there is NO STANDARD.  There is Horea's formalization/framework etc.
```

Comment:
"standard" defined similarly to BIDS is inappropriate, however "standard" as we intended was a redefinition of "complete analysis instructions". 

None of the components of what we did are new, however by using each of the extant components in a way that preserves the ability to re-execute is actually novel because no previous article has been generated, end to end in a way that is reproducible by reviewers (or anyone) without needing the domain knowledge that we needed to perform the reexecution. 

The current "front runners" of re-executable may make code available, but have not automated the process e2e. Because the final result cannot be generated without domain knowledge,  (or at all) we can verifiably demonstrate that every aspect of the data